extensionsGallery-insider.json

```diff
@@ -3822,15 +3822,72 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+					"extensionId": "78",
+					"extensionName": "extraSqlScriptAs",
+					"displayName": "Extra Sql Script As",
+					"shortDescription": "This extension adds several missing options to the context menu of the object tree: Script Table as INSERT, Script Table as UPDATE...",
+					"publisher": {
+					  "displayName": "pacoweb",
+					  "publisherId": "pacoweb",
+					  "publisherName": "pacoweb"
+					},
+					"versions": [
+					  {
+						"version": "1.0.1",
+						"lastUpdated": "31/12/2020",
+						"assetUri": "",
+						"fallbackAssetUri": "fallbackAssetUri",
+						"files": [
+						  {
+							"assetType": "Microsoft.SQLOps.DownloadPage",
+							"source": "https://github.com/pacoweb/extraSqlScriptAs/releases/tag/0.1.0"
+						  },
+						  {
+							"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+							"source": "https://raw.githubusercontent.com/pacoweb/extraSqlScriptAs/master/images/default_icon.png"
+						  },
+						  {
+							"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+							"source": "https://raw.githubusercontent.com/pacoweb/extraSqlScriptAs/master/README.md"
+						  },
+						  {
+							"assetType": "Microsoft.VisualStudio.Code.Manifest",
+							"source": "https://raw.githubusercontent.com/pacoweb/extraSqlScriptAs/master/package.json"
+						  },
+						  {
+							"assetType": "Microsoft.VisualStudio.Services.Content.License",
+							"source": "https://github.com/pacoweb/extraSqlScriptAs/blob/master/LICENSE"
+						  }
+						],
+						"properties": [
+						  {
+							"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+							"value": ""
+						  },
+						  {
+							"key": "Microsoft.VisualStudio.Services.Links.Source",
+							"value": "https://github.com/pacoweb/extraSqlScriptAs"
+						  },
+						  {
+							"key": "Microsoft.VisualStudio.Code.Engine",
+							"value": "*"
+						  }
+						]
+					  }
+					],
+					"statistics": [],
+					"flags": ""
+				  }
 			],
 			"resultMetadata": [
 				{
 					"metadataType": "ResultCount",
 					"metadataItems": [
 						{
 							"name": "TotalCount",
-							"count": 65
+							"count": 66
```

Comment:
This will need to be updated since we just added another new extension - please merge in the latest from releases/extensions and then update the count appropriately.